willeswa/politico-ui
# Bg populate dashboard 164406640

#### What does this PR do?
Populates the user dashboard with logged in user data
### Description of Tasks to be completed?
A user should be able to access their profile page with showing  at least their voting history and  user information 
### How should this be manually tested?
On the pages hosted on GitHub, create a new user account with all valid information including a real passport url
Using the email and password used to register, 
Login into the system
You should be redirected to a dashboard showing your user information a voting history if any
### What are he relevant Pivotal Tracker stories?
https://www.pivotaltracker.com/story/show/164406640
